src/frontend/scss/objects/_course_glimpses.scss

```
@@ -264,6 +264,10 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     top: -3.53rem;
     width: 4.3rem;
     z-index: 0;
+
+    img {
+      max-height: 100%;
+    }
```

In order to cover all layout issues here, I think `object-fit` is a better workaround. What do you think?

```suggestion
    img {
      object-fit: contain;
      object-position: center;
    }
```

---

I didn't know that property...

To make it work, the `img` tag also needs an explicit `height` to have the same effect.
```css
    img {
      object-fit: contain;
      object-position: center;
      height: 4rem;
    }
```

To cover other layout issues, I think it is better to include the width.
```css
    img {
      object-fit: contain;
      object-position: center;
      height: 4rem;
      width: 4rem;
    }
```